looprpc/client.proto

```diff
@@ -823,12 +823,10 @@ message LiquidityParameters {
     uint64 autoloop_budget_sat = 11;
 
     /*
-    The start time for autoloop budget, expressed as a unix timestamp in
-    seconds. If this value is 0, the budget will be applied for all
-    automatically dispatched swaps. Swaps that were completed before this date
-    will not be included in budget calculations.
+    The period over which the autoloop budget is refreshed, expressed in
+    seconds.
     */
-    uint64 autoloop_budget_start_sec = 12;
+    uint64 autoloop_budget_refresh_period_sec = 12;
```

Comment:
This is bad practice. Generally we want to deprecate or delete a field and add a new field instead to no break compatibility.
See: https://protobuf.dev/programming-guides/dos-donts/#dont-re-use-a-tag-number